src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -274,6 +274,18 @@
     type: command
     short-summary: Unregister a provider.
 """
+helps['provider operation'] = """
+    type: group
+    short-summary: get provider operations.
+"""
+helps['provider operation show'] = """
+    type: command
+    short-summary: Get an individual provider's operations.
+"""
+helps['provider operation list'] = """
+    type: command
+    short-summary: Get operations from all providers.
```

Comment:
I thought about the exactly same thing, but backed out once I realized that I was combining a `show` with a `list`, which would cause the command to produce different return types, an object or an array of objects. `list` and `list-all` don't have such difference.